src/components/PeopleTable.tsx

```diff
@@ -1,4 +1,45 @@
-export const PeopleTable = () => {
+import { FC } from 'react';
+import classNames from 'classnames';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  people: Person[];
+  findParent: (parentName: string) => string | JSX.Element;
+};
+
+export const PeopleTable: FC<Props> = ({
+  people,
+  findParent,
+}) => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
+  const { slug } = useParams();
+
+  const sortParams = (sortParam: string) => {
+    if (order && sort) {
```

Comment:
The 'sortParams' function logic is flawed. When 'order' and 'sort' parameters exist, they should not be set to null. Instead, 'order' should be toggled between 'asc' and 'desc'.